homeassistant/components/switch/enocean.py

```diff
@@ -16,35 +16,44 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+CONF_SENDER_ID = 'sender_id'
+
 DEFAULT_NAME = 'EnOcean Switch'
 DEPENDENCIES = ['enocean']
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_ID): vol.All(cv.ensure_list, [vol.Coerce(int)]),
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional(CONF_SENDER_ID, default=[]): vol.All(cv.ensure_list,
+                                                      [vol.Coerce(int)]),
+    vol.Optional("subtype", default=""): cv.string,
```

Comment:
Define a `CONF_SUBTYPE`. There also only fsr61 or permundo. You can set it default to permundo and check the correct value.